docs/rules/ultiduo.md

```diff
@@ -38,10 +47,50 @@ For playoff matches following a best of three (BO3) format, the **higher seeded
 
 The **higher seeded team** is **Team A**. **Maps must be played in the order in which they were picked.**
 
+Using the Captain Discord Match Thread channel 
```

Comment:
Match pick/bans for Ultiduo can and should be done using the pick/bans system on Citadel. The only reason it's not doable for our Sixes seasonal league is we have to add ~11 maps to the league page, but not all divisions are allowed to play all maps, hence why we did it via discord for weekly vetos.